<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>137</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя входного файла :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_fileIn_select">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>20</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_fileRemove">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>удалять входные файлы</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>91</y>
        <width>137</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя исходящего файла :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_fileOut_select">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>90</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Действия при &lt;/p&gt;&lt;p&gt;повторении имени &lt;/p&gt;&lt;p&gt;выходного файла:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_fileOverridePolicy">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Переписать</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Добавить постфикс к имени</string>
       </property>
      </item>
     </widget>
     <widget class="QToolButton" name="toolButton_fileOnceSave">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>129</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Записать 
1 раз</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_forSaveTimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>208</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Записывать файл раз в </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_repeat_foTime">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>216</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>секунд</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_repeat_checkFile">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>248</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>секунд</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Значение 8 байт для бинарной операции</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_binaryOperationValue">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>288</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>254</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Действия :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_action">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AND</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XOR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NOT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox_forOpenTimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверять наличие файла раз в </string>
      </property>
     </widget>
     <widget class="QLabel" name="label__open_file">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>22</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label__save_file">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>93</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Входящий файл</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_income">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>372</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Преобразованный файл</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_output">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>572</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
